ERCS/erc-7786.md

```diff
@@ -103,6 +103,12 @@ This event signals that a would-be sender has requested a message to be sent.
 
 If `outboxId` is present, post-processing MAY be required to send the message through the cross-chain channel.
 
+#### Mandatory and optional attributes
```

Comment:
All attributes should be optional. `read(false)` would be the default.

Reply:
Makes sense. I use the term "Default attributes" instead, in 15c1f1b7dea821026f012451b6191e1c93093c5a. Also following @alxiong's [suggestion](https://github.com/ethereum/ERCs/pull/852#discussion_r1917627298) I used the name `pushBased` instead of `read`.